components/spreadsheet/functions-formulas.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Functions and Formulas
+page_title: Spreadsheet - Functions and Formulas
+description: Learn about all built-in functions supported by the Telerik UI for Blazor Spreadsheet.
+slug: spreadsheet-functions-formulas
+tags: telerik,blazor,spreadsheet
+published: True
+position: 30
+---
+
+# Spreadsheet Functions and Formulas
+
+This article lists the supported built-in functions and formulas by Telerik Spreadsheet for Blazor.
+
+
+## Formulas
+
+A Spreadsheet formula starts with the equal sign `=` and can include:
+
+* References to cells and ranges
+* Constants and literals
+* Operators
+* [Functions](#function-list)
+
+For more information, you can refer to:
+
+* [Microsoft Formula documentation](https://support.microsoft.com/en-us/office/overview-of-formulas-in-excel-ecfdc708-9162-49e8-b993-c311f47ca173)
```

Comment:
Use the corresponding HTML syntax to open the links to external pages in a new tab instead of the same window. You can apply this suggestion to the next item too.
```suggestion
* <a href="https://support.microsoft.com/en-us/office/overview-of-formulas-in-excel-ecfdc708-9162-49e8-b993-c311f47ca173" target="_blank">Microsoft's formulas documentation</a>
```

Reply:
We have almost 3,000 external links in the documentation and less than 10% open in a new tab. I'd say we keep the consistency in the other direction and switch all links at once when we decide to do so.